content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -1317,6 +1330,49 @@ created Pod should be ready without any of its containers crashing, for it to be
 This defaults to 0 (the Pod will be considered available as soon as it is ready). To learn more about when
 a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/pod-lifecycle/#container-probes).
 
+### Pod Replacement Policy
+
+{{< feature-state feature_gate_name="DeploymentPodReplacementPolicy" >}}
+
+{{< note >}}
+This feature is disabled by default. You can enable it by setting the `DeploymentPodReplacementPolicy`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](/docs/reference/command-line-tools-reference/kube-apiserver/)
+and on the [Kube Controller Manager](/docs/reference/command-line-tools-reference/kube-controller-manager/)
+{{< /note >}}
+
+Deployment Pods can become terminating due to deletion, usually during the rollout or scaling of a new
+deployment. Some pods may take a long time to terminate and may consume additional resources during this
+time. As a result, the total number of all pods can temporarily exceed `.spec.replicas`
+(or `.spec.replicas + .spec.strategy.rollingUpdate.maxSurge` for RollingUpdate Deployments).
+A `.spec.podReplacementPolicy` field can be used to control when a Deployment should create new pods.
+
+Policies:
+
+`TerminationStarted`
+: This policy creates replacement Pods when the old Pods start terminating (has a
+`.metadata.deletionTimestamp`). The total number of Deployment Pods can be greater than specified by the
+Deployment's `.spec.replicas` and the DeploymentStrategy. It can increase the speed of rollouts, but it will
+result in a temporarily higher resource consumption.
+
+`TerminationComplete`
+: This policy creates replacement Pods only when the old Pods are fully terminated (reach `Succeeded` or
+`Failed` phase). The old Pods are subsequently removed. The total number of the Deployment Pods is limited by
+the Deployment's `.spec.replicas` and the DeploymentStrategy. It can slow down the speed of rollouts, but
+prevents spikes in resource consumption.
+
+The default behavior when the policy is not specified depends on the DeploymentStrategy:
+* Recreate strategy uses `TerminationComplete` behavior when recreating the deployment,
+  but uses `TerminationStarted` when scaling the deployment.
+* RollingUpdate strategy uses `TerminationStarted` behavior for both rolling out and
+  scaling the deployments.
+
+{{< note >}}
+If you manually delete a Pod, the lifecycle is controlled by the ReplicaSet and the replacement will be created
+immediately (even if the old Pod is still in a Terminating state). The Pod Replacement Policies control only
+Deployment rollouts and scaling.
+{{< /note >}}
```

Comment:
Avoid using too many note shortcodes. Try this instead:
```suggestion
If you manually delete a Pod, the lifecycle of that Pod and its replacement is controlled by the associated ReplicaSet; a replacement will be created immediately. When you delete a Pod manually, and the deleted
Pod was owned by a ReplicaSet, the ReplicaSet creates a new Pod as soon as you trigger deletion (even if the old Pod is still in a Terminating state).
Setting a pod replacement policy for a Deployment only has an effect on:

1. rollouts for changes to a Deployment
2. changes to the scale of that Deployment
```

Reply:
Thanks for the suggestions. As mentioned in https://github.com/kubernetes/website/pull/48370#discussion_r1804830025, I think it is better to just say rollouts. I have also removed the list at the end to keep it more compact and highlight the policies above.